### PR TITLE
Update notice message when no upcoming events

### PR DESCRIPTION
**Pull request by lucatume**
Issue: https://moderntribe.atlassian.net/browse/TEC-3430 

[Screencast](https://drive.google.com/open?id=1DYV7Pozas38jiQ8fRmslBPE-WtTx020u&authuser=luca%40tri.be&usp=drive_fs) 

This PR updates the wording of the notices shown on Month, Day and List View when there are no upcoming events and the date is the default one.

### CODE REVIEW

File: src/Tribe/Views/V2/Messages.php [hunk 1]
```diff
@@ -94,6 +94,10 @@ public static function for_key( $key, ...$values ) {
 				'There were no results found.',
 				'the-events-calendar'
 			),
+			'no_upcoming_events'                 => __(
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Messages.php:97]: :warning: Array double arrow not aligned correctly; expected 15 space(s) between "'no_upcoming_events'" and double arrow, but found 17.

<sub>hash: 66333984b2054f8ba0a2db0725b406b9</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 1 warning(s)

File: src/Tribe/Views/V2/View.php [hunk 2]
```diff
@@ -2243,4 +2244,17 @@ protected function setup_additional_views( array $events = [], array $template_v
...
+
+	/**
+	 * Returns the number of upcoming events in relation to the "now" time.
+	 *
+	 * @since TBD
+	 *
+	 * @return int The number of upcoming events from "now".
+	 */
+	protected function upcoming_events_count() {
+		$from_date = tribe_beginning_of_day( $this->context->get( 'now', date( 'Y-m-d H:i:s' ) ) );
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/View.php:2256]: :no_entry_sign: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.

<sub>hash: f15f34ccaf99715398a2576ff0d1f424</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

File: src/Tribe/Views/V2/Messages.php [hunk 3]
```diff
@@ -94,6 +94,10 @@ public static function for_key( $key, ...$values ) {
 				'There were no results found.',
 				'the-events-calendar'
 			),
+			'no_upcoming_events'               => __(
+				'There are no upcoming events.',
```
Comment (Camwyn) [inline src/Tribe/Views/V2/Messages.php:98]: Can we use the function for the word "events"? At least where we change the code 😉

**Review by Camwyn — CHANGES_REQUESTED** [review]
One change please.

File: src/Tribe/Views/V2/Messages.php [hunk 4]
```diff
@@ -94,6 +94,10 @@ public static function for_key( $key, ...$values ) {
 				'There were no results found.',
 				'the-events-calendar'
 			),
+			'no_upcoming_events'               => __(
```
Comment (jesseeproductions) [inline src/Tribe/Views/V2/Messages.php:97]: Looks good, can you use the function for the word events here? ( plural lowercase )

**Review by jesseeproductions — COMMENTED** [review]
Looks good, just a change for the word events.

**Review by Camwyn — APPROVED** [review]

**Pull request merged**